### ISSUE TITLE
Suggestion for improved figure/subplot syntax

### ISSUE THREAD

**Issue by markusrobertjonsson** [learningsimulator/learningsimulator#165]
Suggestion for improved `@figure/@subplot` syntax:

- Specify subplot grid in `@figure`
- Rename `@subplot` to @panel and use default subplot numbering
(- Easier specification of axis labels (xlabel, ylabel) and axis limits (xlim, ylim) by moving them outside of the {mpl options})
(- More compact specification of legend by adding the word "legend" to @panel)
```
@figure(2,3) myFigTitle
@panel myPanelTitle legend {mpl options}
subject: all
@pplot red->response  # Is plotted in subplot 1,1 (that will have a legend)

@panel myPanelTitle {mpl options}
subject: average
@pplot blue->response  # Is plotted in subplot 1,2 (that will not have a legend)
@pplot yellow->response  # Is plotted in subplot 1,2 (that will not have a legend)
...
```

Consider adding the wildcard syntax for plotting commands as suggested in issue #96 in conjunction with this.

**Comment by markusrobertjonsson** [comment]
Think through if `xlabel` and `ylabel` are the best choices to put outside {mpl_options}. How about `xlim`, `ylim`?

**Comment by markusrobertjonsson** [comment]
Fixed in version 1.1.
